x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/rule_management/logic/import/rule_source_importer/rule_source_importer.ts

```diff
@@ -165,11 +176,18 @@ export class RuleSourceImporter implements IRuleSourceImporter {
     }, {});
   }
 
+  private async fetchInstalledRulesByIds(ruleIds: string[]): Promise<Record<string, RuleResponse>> {
```

Comment:
nit: Add `Map` to the name for clarity 

```suggestion
  private async fetchInstalledRulesMapByIds(ruleIds: string[]): Promise<Record<RuleSignatureId, RuleResponse>> {
```